include/ck_tile/ops/gemm/pipeline/block_gemm_pipeline_agmem_bgmem_creg_universal_policy.hpp

```diff
@@ -0,0 +1,421 @@
+                make_tuple(
+                    make_pass_through_transform(number<kKThreadWrite / kfold / kKThreadReadPerm>{}),
+                    make_pass_through_transform(number<kK0PerThreadWrite>{}),
+                    make_xor_transform(make_tuple(number<kKThreadReadPerm * kN1>{},
+                                                  number<kfold * kN0 / npair>{})),
+                    make_pass_through_transform(number<npair>{}),
+                    make_pass_through_transform(kK1)),
+                make_tuple(
+                    sequence<0>{}, sequence<1>{}, sequence<2, 3>{}, sequence<4>{}, sequence<5>{}),
+                make_tuple(
+                    sequence<0>{}, sequence<1>{}, sequence<2, 3>{}, sequence<4>{}, sequence<5>{}));
+
+            constexpr auto b_lds_block_desc_unmerged = transform_tensor_descriptor(
+                b_lds_block_desc_permuted,
+                make_tuple(
+                    make_pass_through_transform(number<kKThreadWrite / kfold / kKThreadReadPerm>{}),
+                    make_pass_through_transform(number<kK0PerThreadWrite>{}),
+                    make_unmerge_transform(make_tuple(number<kKThreadReadPerm>{}, number<kN1>{})),
+                    make_unmerge_transform(make_tuple(number<kfold>{}, number<kN0 / npair>{})),
+                    make_pass_through_transform(number<npair>{}),
+                    make_pass_through_transform(kK1)),
+                make_tuple(sequence<0>{},
+                           sequence<1>{},
+                           sequence<2>{},
+                           sequence<3>{},
+                           sequence<4>{},
+                           sequence<5>{}),
+                make_tuple(sequence<1>{},
+                           sequence<2>{},
+                           sequence<0, 3>{},
+                           sequence<4, 5>{},
+                           sequence<6>{},
+                           sequence<7>{}));
+
+            constexpr auto b_lds_block_desc_n_k = transform_tensor_descriptor(
+                b_lds_block_desc_unmerged,
+                make_tuple(make_merge_transform_v3_division_mod(
+                               make_tuple(number<kKThreadReadPerm>{},
+                                          number<kKThreadWrite / kfold / kKThreadReadPerm>{},
+                                          number<kfold>{},
+                                          number<kK0PerThreadWrite>{})),
+                           make_merge_transform_v3_division_mod(
+                               make_tuple(number<kN0 / npair>{}, number<npair>{}, number<kN1>{})),
+                           make_pass_through_transform(kK1)),
+                make_tuple(sequence<0, 1, 4, 2, 7>{}, sequence<5, 6, 3>{}),
+                make_tuple(sequence<1>{}, sequence<0>{}));
+
+            return b_lds_block_desc_n_k;
+        }
+    }
+
+    template <typename Problem>
+    CK_TILE_HOST_DEVICE static constexpr ck_tile::index_t GetSmemSizeA()
```

Comment:
You don't need to use namespace resolution for symbol which is visible in the namespace you're currently in. 
Please change this also in other occurrences.

```suggestion
    CK_TILE_HOST_DEVICE static constexpr index_t GetSmemSizeA()
```